runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java

```diff
@@ -195,6 +195,7 @@ private CoordinateClueInfo(@Nonnull String directions, boolean lightRequired, Va
 		.put(new WorldPoint(2872, 3937, 0), new CoordinateClueInfo("Weiss."))
 		.put(new WorldPoint(2484, 4016, 0), new CoordinateClueInfo("Northeast corner of the Island of Stone."))
 		.put(new WorldPoint(2222, 3331, 0), new CoordinateClueInfo("Prifddinas, west of the Tower of Voices"))
+		.put(new WorldPoint(3560, 3987, 0), new CoordinateClueInfo("Lithkren. If unlocked, Digsite pendant teleport, Otherwise take rowboat directly west of Mushroom Meadow Mushtree."))
```

Comment:
```suggestion
		.put(new WorldPoint(3560, 3987, 0), new CoordinateClueInfo("Lithkren. Digsite pendant teleport if unlocked, otherwise take rowboat from west of Mushroom Meadow Mushtree."))
```